SWLOR.Game.Server/Feature/RecipeDefinition/EngineeringRecipeDefinition/StarshipRecipes.cs

```diff
@@ -48,12 +48,12 @@ private void Tier1()
                 .Component("fiberp_ruined", 3)
                 .Component("elec_ruined", 3);
 
-            // Ship Missiles x3
-            _builder.Create(RecipeType.Missile3, SkillType.Engineering)
+            // Ship Missiles x5
```

Comment:
I should have caught this on the initial submission, but these ammo recipes shouldn't be in the StarshipRecipes.cs class. I think they should be moved into either the ModuleRecipes.cs file or their own StarshipAmmo.cs class.

Reply:
Wont moving them over directly cause issues to crafted things though?

Reply:
The class files have no bearing on how the data gets stored. You can freely move them between the files - they're just a way of grouping recipes together. If I need to find the recipes for missiles, one of the last places I'll think to look is in the StarshipRecipes file. It's just a copy/paste job of moving it out of this file and into the StarshipModules or StarshipAmmo class, whichever you end up doing. I don't have a preference - but it shouldn't be in here with the starship recipes

Reply:
Okay no problems moving it, I'll make it a StarshipAmmo, it makes the most sense being as we now have what 4 ammo types for ships.